tools/SetupFlow/DevHome.SetupFlow/Views/SummaryView.xaml

```diff
@@ -210,10 +210,10 @@
                                     <RowDefinition/>
                                     <RowDefinition/>
                                 </Grid.RowDefinitions>
-                                <TextBlock Text="Error logs" Grid.Row="0" Grid.Column="0" Margin="0, 5, 0, 0" Foreground="{StaticResource TextFillColorPrimary}"/>
+                                <TextBlock Text="Error logs" Grid.Row="0" Grid.Column="0" Margin="0, 5, 0, 0"/>
                                 <StackPanel Orientation="Horizontal" Spacing="5" Grid.Row="1" Grid.Column="0">
-                                    <TextBlock Text="{x:Bind ViewModel.FailedTasks.Count}" Foreground="{StaticResource TextFillColorSecondary}"/>
-                                    <TextBlock Text="installation errors" Foreground="{StaticResource TextFillColorSecondary}"/>
+                                    <TextBlock Text="{x:Bind ViewModel.FailedTasks.Count}" Foreground="{ThemeResource TextFillColorSecondary}"/>
+                                    <TextBlock Text="installation errors" Foreground="{ThemeResource TextFillColorSecondary}"/>

```

Comment:
Thanks for this @hamnarauf.

Looks good, but since you're doing this already, I see that on line 230 in this file we're also using StaticResource instead of the ThemeResource for the foreground of the 'MessageToShow' textblock.  Would you be able to update that as well? 

Thanks again!

Reply:
Made `MessageToShow` use `ThemeResource`